canonical-web-and-design/charmhub.io
# Add list and listing pages

## Done

- Add `/charms-and-bundles` page
- Add `/<charm_name>/listing` page (only html and css for now)
- Swap `/docs` content to use the Snapcraft one

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045/charms-and-bundles & http://0.0.0.0:8045/<charm_name>/listing
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [designs](https://app.zeplin.io/project/5d66544639bbe3139a99e168/dashboard?sid=5d6656f0641a109bd392ba40)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/charmhub.io/issues/141
